MetadataExtractor/Formats/Exif/ExifDescriptorBase.cs

```diff
@@ -751,11 +751,11 @@ private static string FormatCFAPattern(int[] pattern)
                     ret.Append("Unknown");  // indicated pattern position is outside the array bounds
 
                 if ((pos - 2) % pattern[1] == 0)
-                    ret.Append(",");
+                    ret.Append(',');
                 else if (pos != end - 1)
                     ret.Append("][");
             }
-            ret.Append("]");
+            ret.Append(']');
```

Comment:
The changes of this kind could be pulled out to another PR, which I'd merge with only a quick review.